### ISSUE TITLE
twin_axis not showing label correctly in version 0.12.9

### ISSUE THREAD

**Issue by wqp89324** [bokeh/bokeh#7060]
I recently upgraded from 0.12.6 to 0.12.9, then I found there is an issue with the LinearAxis object, which would not display the label correctly. 
I attach some code, and screen shots of expected and bug behavior, one from 12.6, the other from 12.9. In 12.9, you can see the y axis label "x" is missing. 

```
from numpy import pi, arange, sin, linspace
from bokeh.models import LinearAxis, Range1d
from bokeh.plotting import figure, show, output_file

x = arange(-2*pi, 2*pi, 0.1)
y = sin(x)
y2 = linspace(0, 100, len(x))
p = figure(x_range=(-6.5, 6.5), y_range=(-1.1, 1.1), min_border=80)
p.circle(x, y, fill_color="red", size=5, line_color="black")
p.extra_y_ranges['foo'] = Range1d(0, 100)
p.circle(x, y2, fill_color="blue", size=5, line_color="black", y_range_name="foo")
p.add_layout(LinearAxis(y_range_name="foo",
                        axis_label = "x"), 'left')
output_file("twin_axis.html", title="twin_axis.py example")
show(p)

```
![0 12 6](https://user-images.githubusercontent.com/8864839/31632009-1d426154-b281-11e7-91ea-3cd012e94b4c.PNG)
![0 12 9](https://user-images.githubusercontent.com/8864839/31632010-1d58cb24-b281-11e7-8cfb-35afe534c643.PNG)

**Comment by mattpap** [comment]
Label is missing, but canvas layout is correct (the second screenshot may indicate otherwise).

**Comment by wqp89324** [comment]
@mattpap @bryevdv 
I just upgraded from 12.0.6 to 12.0.10, but I still have this problem: the label on twin axis is missing. Is this going be to fixed in version 12.0.11?

**Comment by bryevdv** [comment]
@wqp89324 Yes you can see the milestone `0.12.11` listed above (this was merged after `0.12.10` was released)
